src/actionCreators/index.js

```diff
@@ -37,18 +38,29 @@ const apiPollTimeout = 3000;
 const maxStatusCheckCount = (120 * 1000) / apiPollTimeout;
 let statusCheckCount = 0;
 
-export const createSigningJob = files => dispatch => {
+const authenticationHeaderVaule = () => {
+  try {
+    const token = JSON.parse(localStorage.getItem('auth')).id_token.bearer;
+    return `Bearer ${token}`;
```

Comment:
I'll add this to the wishlist of things to add to the auth manager, this really should be a helper method that you utilize